t/venom/lib/pf_api_service_action.yml

```diff
@@ -1,21 +1,21 @@
 executor: pf_api_service_action
 input:
-  service: {}
-  action: {}
+  action: ""
+  body: {}
+  method: ""
+  service: ""
+  status_code: ""
+  time_to_sleep: 5
 steps:
-- type: get_login_token
-
-- type: http
-  method: POST
-  url: '{{.pfserver_webadmin_url}}/api/v1/service/{{.input.service}}/{{.input.action}}'
-  ignore_verify_ssl: true
-  headers:
-    "Authorization": "{{.result.token}}"
-    "Content-Type": "application/json"
-  assertions:
-    - result.statuscode ShouldEqual 200
-  output: 
-     postjson: "{{.result.bodyjson}}"
+- type: pf_api_action
+  body: '{{.input.body}}'
+  status_code: "{{.input.status_code}}"
+  method: "{{.input.method}}"
+  time_to_sleep: "{{.input.time_to_sleep}}"
+  url: 'service/{{.input.service}}/{{.input.action}}'
+  vars:
+    service_temp:
+       from: result.pf_api_action_
```

Comment:
Not able to understand why there is a leading `_`.

I see it is also present in `pf_api_action` executor (under `output`)

Reply:
The leading is due to: https://github.com/ovh/venom#user-defined-executors
```
Notice the variable alljson. All variables declared in output are automatically converted in a json format with the suffix json. In the example above, two implicit variables are available: displayjson.hello and alljson.
```
At the end the output has a `json` added so to be more readable instead of pf_api_actionjson i have preferred pf_api_action_json so it needs a tail _.